Index: mfUtils/include/mfUtils/Utils/Utils.cpp
===================================================================
--- mfUtils/include/mfUtils/Utils/Utils.cpp	(revision 16)
+++ mfUtils/include/mfUtils/Utils/Utils.cpp	(working copy)
@@ -22,5 +22,10 @@
 			return LPCTSTR(str);
 #endif // UNICODE		
 		}
+
+		bool MFCUtils::IsEqual(float a, float b, float epsilon)
+		{
+			return (fabs(a - b) < epsilon);
+		}
 	}
 }
\ No newline at end of file
Index: mfUtils/include/mfUtils/Utils/Utils.h
===================================================================
--- mfUtils/include/mfUtils/Utils/Utils.h	(revision 16)
+++ mfUtils/include/mfUtils/Utils/Utils.h	(working copy)
@@ -10,8 +10,9 @@
 		class MFCUtils
 		{
 		public:
-			static LPCTSTR ToLPCTSTR(const std::string& str);
-			static LPCTSTR ToLPCTSTR(const char* str);
+			static LPCTSTR	ToLPCTSTR(const std::string& str);
+			static LPCTSTR	ToLPCTSTR(const char* str);
+			static bool		IsEqual(float a, float b, float epsilon = FLT_EPSILON);
 		};
 	}
 }
Index: mfUtils/prj/mfUtils.vcxproj
===================================================================
--- mfUtils/prj/mfUtils.vcxproj	(revision 16)
+++ mfUtils/prj/mfUtils.vcxproj	(working copy)
@@ -110,7 +110,7 @@
       <AdditionalIncludeDirectories>..\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PrecompiledHeader>Use</PrecompiledHeader>
       <PrecompiledHeaderFile>mfUtils/pch.h</PrecompiledHeaderFile>
-      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;_UNICODE;UNICODE;_AFXDLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
       <SubSystem>Console</SubSystem>
@@ -125,7 +125,7 @@
       <AdditionalIncludeDirectories>..\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PrecompiledHeader>Use</PrecompiledHeader>
       <PrecompiledHeaderFile>mfUtils/pch.h</PrecompiledHeaderFile>
-      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;_UNICODE;UNICODE;_AFXDLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
       <SubSystem>Console</SubSystem>
@@ -142,7 +142,7 @@
       <AdditionalIncludeDirectories>..\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PrecompiledHeader>Use</PrecompiledHeader>
       <PrecompiledHeaderFile>mfUtils/pch.h</PrecompiledHeaderFile>
-      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;_UNICODE;UNICODE;_AFXDLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
       <SubSystem>Console</SubSystem>
@@ -161,7 +161,7 @@
       <AdditionalIncludeDirectories>..\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PrecompiledHeader>Use</PrecompiledHeader>
       <PrecompiledHeaderFile>mfUtils/pch.h</PrecompiledHeaderFile>
-      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;_UNICODE;UNICODE;_AFXDLL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
     </ClCompile>
     <Link>
       <SubSystem>Console</SubSystem>
